packages/public/server/src/module/Entity/src/Entity/Controller/EntityController.php

```diff
@@ -86,7 +86,41 @@ public function unrevisedAction()
             }
         }
 
-        $view = new ViewModel(['revisionsBySubject' => $revisionsBySubject]);
+        return $revisionsBySubject;
+    }
+
+    protected function getReviewHelpLinks()
+    {
+        $instance = $this->getInstanceManager()->getInstanceFromRequest();
+
+        if ($instance->getLanguage()->getCode() == "de") {
+            $reviewHelpUrl = "/140479";
+        } else {
+            $reviewHelpUrl = "https://docs.google.com/document/d/1p03xx2KJrFw8Mui4-xllvSTHcEPi8G1bdC8rGXcH6f8/edit";
+        }
+
+        $helpLinks = [
+            [ $reviewHelpUrl, "Guideline for reviewing"],
+            [ "/discussions", "List of all discussions"],
+            [
+                "https://community.serlo.org/channel/feedback-requests",
+                "Channel #feedback-requests in RocketChat",
+            ],
+            [
+                "https://docs.google.com/forms/d/e/1FAIpQLSfMjWIZZq2_AoHbqNv3AOEjQRBwA8qEZIMJpk5l0vX7w2nwnQ/viewform",
+                "Questionnaire for reviewers",
+            ],
+        ];
```

Comment:
It feels like that should be translatable / instance-aware. RocketChat is mostly de currently and the feedback form is also in German. Also: possibly hide stuff depending on role. Similarly to above: it feels kinda sketchy that we redirect community reviewers to Google Forms.

Reply:
RocketChat will be also used by non-german communities (see for example https://community.serlo.org/channel/tamil ).

@yolo-bird : Do you have on your todo list to translate the questionnaire or shall we include this link only in the German Serlo version? Do you want to hide links for some roles?